hooks/opositeSideGenerator.js

```diff
@@ -0,0 +1,19 @@
+module.exports = register => {
+    register('generate:before', generator => {
+        if (generator.templateParams && generator.templateParams['generateOppositeSide'] === 'true') {
+            console.log("Opposite side generation mode is enabled");
```

Comment:
left by accident?

Reply:
No, it was intentionally, because it's very specific mode, i decided to leave a message in the log.